.teamcity/common/Os.kt

```diff
@@ -16,26 +16,35 @@
 
 package common
 
-val killAllGradleProcessesUnixLike = """
-    free -m
-    ps aux | egrep 'Gradle(Daemon|Worker)'
-    ps aux | egrep 'Gradle(Daemon|Worker)' | awk '{print ${'$'}2}' | xargs kill -9
-    free -m
-    ps aux | egrep 'Gradle(Daemon|Worker)' | awk '{print ${'$'}2}'
-""".trimIndent()
+private const val killAllGradleProcessesUnixLike = """
+free -m
+ps aux | egrep 'Gradle(Daemon|Worker)'
+ps aux | egrep 'Gradle(Daemon|Worker)' | awk '{print ${'$'}2}' | xargs kill -9
+free -m
+ps aux | egrep 'Gradle(Daemon|Worker)' | awk '{print ${'$'}2}'
+"""
 
-val killAllGradleProcessesWindows = """
-    wmic OS get FreePhysicalMemory,FreeVirtualMemory,FreeSpaceInPagingFiles /VALUE
-    wmic Path win32_process Where "name='java.exe'"
-    wmic Path win32_process Where "name='java.exe' AND CommandLine Like '%%%%GradleDaemon%%%%'" Call Terminate
-    wmic Path win32_process Where "name='java.exe' AND CommandLine Like '%%%%GradleWorker%%%%'" Call Terminate
-    wmic OS get FreePhysicalMemory,FreeVirtualMemory,FreeSpaceInPagingFiles /VALUE
-    wmic Path win32_process Where "name='java.exe'"
-""".trimIndent()
+private const val killAllGradleProcessesWindows = """
+wmic OS get FreePhysicalMemory,FreeVirtualMemory,FreeSpaceInPagingFiles /VALUE
+wmic Path win32_process Where "name='java.exe'"
+wmic Path win32_process Where "name='java.exe' AND CommandLine Like '%%%%GradleDaemon%%%%'" Call Terminate
+wmic Path win32_process Where "name='java.exe' AND CommandLine Like '%%%%GradleWorker%%%%'" Call Terminate
+wmic OS get FreePhysicalMemory,FreeVirtualMemory,FreeSpaceInPagingFiles /VALUE
+wmic Path win32_process Where "name='java.exe'"
+"""
 
-enum class Os(val agentRequirement: String, val ignoredSubprojects: List<String> = emptyList(), val androidHome: String, val killAllGradleProcesses: String) {
+enum class Os(
+    val agentRequirement: String,
+    val ignoredSubprojects: List<String> = emptyList(),
+    val androidHome: String,
+    val killAllGradleProcesses: String,
+    val perfTestWorkingDir: String = "%teamcity.build.checkoutDir%"
+) {
     LINUX("Linux", androidHome = "/opt/android/sdk", killAllGradleProcesses = killAllGradleProcessesUnixLike),
-    WINDOWS("Windows", androidHome = """C:\Program Files\android\sdk""", killAllGradleProcesses = killAllGradleProcessesWindows),
+    WINDOWS("Windows",
+        androidHome = """C:\Program Files\android\sdk""",
+        killAllGradleProcesses = killAllGradleProcessesWindows,
+        perfTestWorkingDir = "P:/"),
```

Comment:
```suggestion
        perfTestWorkingDir = "P:\\"),
```

Why not end the drive with a `\`?

Reply:
I usually use `/` to avoid windows backslash issues. What's wrong with `/`? It doesn't require escaping.

Reply:
Referencing a Windows drive with `/` seems very strange to me. As long as it works I don't feel very strongly about it.